api/swagger.yaml

```diff
@@ -0,0 +1,123 @@
+---
+swagger: '2.0'
+info:
+  title: Lovebot Game Runner
+  description: API for registering player AIs and running game simulations of Loveletter
+  version: '1.0'
+consumes:
+  - application/json
+produces:
+  - application/json
+basePath: "/api"
+paths:
+  "/runGame":

```

Comment:
Even though I merged it, I probably should have been more anal about this one.  Lets keep the verbs out of the API.  This should probably be tournament, as that is the resource we're creating.  We could save off the tournament data onto s3, and then it would really make a lot of sense to be posting a tournament